queries.sql

```diff
@@ -8,3 +8,69 @@
       SELECT * FROM animals WHERE neutered=true;
       SELECT * FROM animals WHERE name NOT IN ('Gabumon');
       SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.13 ;
+
+
+
+
+      
+-- Query and Update animals table
+
+BEGIN;
+
+UPDATE animals SET species='unspecified';
+
+ROLLBACK;
+
+
+BEGIN;
+
+UPDATE animals
+SET species ='digimon'
+WHERE name like '%mon';
+
+
+UPDATE animals
+SET species ='pokemon'
+WHERE name NOT like '%mon';
+
+COMMIT;
+
+BEGIN;
+
+DELETE FROM animals;
+
+ROLLBACK;
+
+
+BEGIN;
+
+SAVEPOINT SP1;
+
+DELETE FROM animals WHERE date_of_birth > '2022-01-01';
+
+SAVEPOINT SP2;
+
+UPDATE animals SET weight_kg=weight_kg * (-1);
+
+ROLLBACK TO SP1;
+
+UPDATE animals SET weight_kg = weight_kg * -1 WHERE weight_kg < 0;
+
+COMMIT;
```

Comment:
- Good job done so far, but you have not correctly run this transaction, please take your time to re-run all transactions and make sure you roll back to the correct savepoint. If your transactions were correctly run, you should have only 10 animals in your database. Please fix this 🙏